Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -135,14 +137,17 @@
               <button mat-button [routerLink]="['/rooms']" i18n="@@navigationRooms">
                 Rooms
               </button>
-              <button mat-icon-button [matMenuTriggerFor]="personalmenu">
-                <mat-icon
-                  matBadge="!"
-                  matBadgeColor="accent"
-                  matBadgeSize="small"
-                  [matBadgeHidden]="!releaseInfoService.newUpdate"
-                  >person</mat-icon
-                >
+              <button
+                mat-icon-button
+                matBadge="!"
+                matBadgeColor="accent"
+                matBadgeSize="small"
+                [matBadgeHidden]="releaseInfoService.newUpdate"

```

Comment:
```suggestion
                [matBadgeHidden]="!releaseInfoService.newUpdate"
```

Reply:
I forgot, undo that
--> fixed that in the commit